api/models.py

```diff
@@ -243,6 +243,7 @@ class Event(models.Model):
     """ A disaster, which could cover multiple countries """
 
     name = models.CharField(max_length=100)
+    slug = models.CharField(max_length=50, default=None, unique=True, null=True)
```

Comment:
this will need a `required=False` else the field is still required in the Admin interface.

Reply:
> this will need a `required=False` else the field is still required in the Admin interface.

Dunno about the `required` attribute, but `blank=True` would work for sure.